<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtKalmanFiltererMainDialog</class>
 <widget class="QDialog" name="QtKalmanFiltererMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KalmanFilterer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
  background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}

QWidget { font: bold, &quot;Courier New&quot;, courier, monospace }
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="area_left">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroll_area_parameters_layout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>250</width>
         <height>275</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="button_start">
          <property name="text">
           <string>&amp;Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="box_show_calculation">
          <property name="text">
           <string>Show calculation</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="box_show_graphs">
          <property name="text">
           <string>Show graphs</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="box_show_statistics">
          <property name="text">
           <string>Show statistics</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="box_show_table">
          <property name="text">
           <string>Show table</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tab_main">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="layout_context">
       <attribute name="title">
        <string>&amp;1. Context</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tab_context">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_web_view_context">
           <attribute name="title">
            <string>Rendered HTML (read-only)</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>web_view_context disabled, due to Qt5 dependency problems</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_plaintext_context">
           <attribute name="title">
            <string>Plaintext HTML</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="edit_context">
              <property name="font">
               <font>
                <family>Courier New,courier,monospace</family>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_calculation">
       <attribute name="title">
        <string>&amp;2. Calculation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scroll_area_calculation">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scroll_area_calculation_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>80</width>
             <height>16</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_graph">
       <attribute name="title">
        <string>&amp;3. Graphs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="3">
         <widget class="QScrollArea" name="scroll_area_graph">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scroll_area_graph_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>80</width>
             <height>16</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="button_save_graph">
          <property name="text">
           <string>Save graphs to file</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_statistics">
       <attribute name="title">
        <string>&amp;4. Statistics</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scroll_area_statistics">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scroll_area_statistics_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>80</width>
             <height>16</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_table">
       <attribute name="title">
        <string>&amp;5. Table</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scroll_area_table">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scroll_area_table_contents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>80</width>
             <height>16</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
